pages/linux/mogrify.md

```diff
@@ -0,0 +1,24 @@
+# mogrify
+
+> Edit and transform an image by overwriting the original file.
+> More information: <https://manned.org/mogrify>.
+
+- Join images into a single multi-image file:
+
+`mogrify -adjoin {{input_file}} {{input_file}}`
```

Comment:
```suggestion
`mogrify -adjoin {{path/to/file1 path/to/file2 ...}}`
```

Mandatory fix: [path format is incorrect](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths).
Mandatory fix: command syntax is incorrect: not just 2 arguments expected.